jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/ButtonToggleTests.java

```diff
@@ -0,0 +1,64 @@
+package io.github.epam.angular.tests.elements.complex;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static io.github.com.StaticSite.angularPage;
+import static io.github.com.pages.sections.ButtonToggleSection.*;
+import static io.github.epam.site.steps.States.shouldBeLoggedIn;
+
+public class ButtonToggleTests extends TestsInit {
+
+    private final String ITALIC = "italic";
+    private final String BOLD = "bold";
+    private final String UNDERLINE = "underline";
+    private final String LEFT = "left";
+    private final String CENTER = "center";
+    private final String RIGHT = "right";
+
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        angularPage.shouldBeOpened();
+    }
+
+    @Test
+    public void verifyButtonToggle() {
+        basicButtonToggle.is().displayed();
+        basicButtonToggle.is().enabled();
+        basicButtonToggle.clickButtonToggleByValue(ITALIC);
+        basicButtonToggle.is().assertButtonToggleIsSelected(ITALIC);
+        basicButtonToggle.is().assertButtonToggleButtonIsPressed(ITALIC);
+        basicButtonToggle.is().assertButtonToggleButtonHasText(ITALIC);
```

Comment:
It's too much checks in one test. Could you please split them into several parts.